.github/workflows/centos-stream-9.yml

```diff
@@ -690,3 +690,88 @@ jobs:
           path: |
             *.json
             *.log
+
+  pre-edge-vsphere-cs9:
+    needs: pr-info
+    if: ${{ needs.pr-info.outputs.allowed_user == 'true' && github.event.issue.pull_request &&
+            (endsWith(github.event.comment.body, '/test-cs9') ||
+            endsWith(github.event.comment.body, '/test-cs9-vsphere')) }}
+    runs-on: ubuntu-latest
+    env:
+      STATUS_NAME: edge-vsphere-installer-cs9
```

Comment:
What does the `installer` mean in STATUS_NAME. I don't think it's reasonable.

Reply:
removed installer